File: backend/models/Wedding.ts
import mongoose from "mongoose";

const Schema = mongoose.Schema;

const WeddingSchema = new Schema({
    weddingUUID: {
        type: Schema.Types.String,
        required: true
    },
    name: {
        type: Schema.Types.String,
        required: true
    },
    location: {
        type: Schema.Types.String,
        required: true
    },
    images: [{
        type: Schema.Types.String
    }], 
    date: {
        type: Schema.Types.String,
        required: true
    }
})

export const Wedding = mongoose.model("Wedding", WeddingSchema)

File: WeddingPlannerBuddy/Model/Wedding.swift
//
//  Wedding.swift
//  WeddingPlannerBuddy
//
//  Created by Alexia Aldea on 26.12.2024.
//

import Foundation

struct Wedding: Codable {
    let id: String
    let name: String
    let date: String
    let location: String
    let images: [String]
    let weddingUUID: String
}
